imports/ui/Results.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { Component } from 'react';
+import "./css/Results.css"; // css files
+import SearchBar from './SearchBar.jsx';
+import ResultsDisplay from './ResultsDisplay.jsx';
+
+
+/*
+  Results Component. Short description if needed.
+
+  Identify as one of the following components:
+  Simple styling: mainly renders HTML and CSS,
+  Container: combines multiple components into a single feature
+  View: top-level component accessed by a URL endpoint defined by the router in main.jsx
+
+  Include a breif description of the component's purpose, where it falls in the
+  component tree, and any inportant information it accesses or modifies.
+  Include the route for View components.
+*/
+
+export class Results extends Component {
+  constructor(props) {
+    super(props);
+    _isMounted = false;
+    this.state = {
+      courseList: [],
+      query: '',
+    };
+
+    this.updateQuery = this.updateQuery.bind(this);
+
+  }
+
+  updateQuery(event) {
+    // trim the query to remove trailing spaces      
+    this.setState({ query: event.target.value.trim() });
+    //Session to be able to get info from this.state.query in withTracker
+    Session.set('querySession', this.state.query);
+  }
+
+  componentDidMount() {
+    this._isMounted = true;
+    Meteor.call("getCoursesByFilters", {
+      classRating: 4.5
+    }, (err, courseList) => {
+      if (!err && courseList.length != 0 && this._isMounted) {
+        // Save the Class object that matches the request
+        this.setState({
+          courseList: courseList
+        });
+      }
+      else {
+        this.setState({
+          courseList: []
+        });
+      }
+    });
+  }
+
+  componentWillReceiveProps(nextProps) {
```

Comment:
This lifecycle method is deprecated in React in the latest version: https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops
We should avoid it.